lib/rules/landmark-at-least-one-main.json

```diff
@@ -0,0 +1,16 @@
+{
+  "id": "landmark-at-least-one-main",
+  "selector": "html",
+  "tags": [
+    "cat.keyboard"
```

Comment:
This should be tagged as `best-practice`, and potentially `cat.navigation` instead of `cat.keyboard`.